app/views/users/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<html>
+  <body>
+    <h1>List of Users</h1>
+  </body>
```

Comment:
- [ OPTIONAL ] I suggest this user show file should have a placeholder "Details of a user" since it is responsible for displaying the details of a given user.